keyboards/7liang/tp32/rules.mk

```diff
@@ -0,0 +1,25 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE 		= yes		# Enable Bootmagic Lite
+MOUSEKEY_ENABLE 		= yes       # Mouse keys
+EXTRAKEY_ENABLE 		= yes       # Audio control and System control
+CONSOLE_ENABLE 			= no        # Console for debug
+COMMAND_ENABLE 			= no        # Commands for debug and configuration
+NKRO_ENABLE 			= yes       # Enable N-Key Rollover
+BACKLIGHT_ENABLE 		= no        # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE 		= yes       # Enable keyboard RGB underglow
+AUDIO_ENABLE 			= no        # Audio output
+SLEEP_LED_ENABLE		= no
+
+PS2_MOUSE_ENABLE 		= yes 
+PS2_USE_USART 			= yes
+# PS2_USE_INT = yes
+
+DEFAULT_FOLDER = 7liang/tp32/v1
```

Comment:
This needs to be removed, or a rules.mk file needs to be added to the v1 folder.

Reply:
ok, thanks

Reply:
```suggestion
DEFAULT_FOLDER = 7liang/tp32/v1

```